x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/CollectedProfiles.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.plugin;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.compute.operator.DriverProfile;
+import org.elasticsearch.xpack.esql.planner.PlannerProfile;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Holds the collection of driver profiles and query planning profiles from the individual nodes processing the query.
+ */
+public class CollectedProfiles implements Writeable {
```

Comment:
Should this just *be* `Profile`?  Both are writable, and they store the same things.